lib/generator/repository.rb

```diff
@@ -35,6 +35,10 @@ def create_tests_file
     def exercise_name
       @exercise_name ||= slug.tr('-', '_')
     end
+
+    def test_case_name
+      exercise_name + '_case'
```

Comment:
This is the wrong place for this method.
If it matters what the filename is it should somewhere under Generator::Files